packages/canary/src/components/text.tsx

```diff
@@ -33,7 +33,7 @@ const textVariants = cva('text-base', {
       both: ''
     },
     truncate: {
-      true: 'truncate text-ellipsis text-nowrap'
+      true: 'truncate'
```

Comment:
This class already contains the necessary properties such as `text-ellipsis text-nowrap`, so they aren't necessary

![Google Chrome 2024-11-11 17 06 06](https://github.com/user-attachments/assets/5ffd1a29-99c6-4701-9b20-98d933a23e9d)